app/read_licenses.py

```diff
@@ -45,7 +45,16 @@ def split_license_data(text):
 
 
 def parse_server_info(lines):
-    return parse("{:^}:\n{}: LICENSE SERVER {:w} (MASTER) V11.16.2{:^}", lines, case_sensitive=False)
+    """
+    Returns an iterable parse.Result object. The parsed word({:w}) should indicate the server status (UP/DOWN). 
+    If the parsing fails (returns None) the parsing in this function may need to be adjusted.
+    """
+    p = parse("{:^}:\n{}: LICENSE SERVER {:w} (MASTER) V{}\n{:^}", lines, case_sensitive=False)
+    if p == None:
+        p = parse("{:^}:\n{}: LICENSE SERVER {:w} V{}\n{:^}", lines, case_sensitive=False)
+    if p == None:
+        logger.info('Server status parse failed. Check read_licenses.parse_server_info against lmutil.exe output.')
+    return p
```

Comment:
Any chance you can update the [data](https://github.com/ishiland/ArcGIS-License-Tracker/tree/master/tests/data) folder with some mock outputs of the what the newest lmutil.exe generates? From my understanding the lmutil.exe output formatting has changed a bit and it probably only makes sense to test this PR against the latest version.

Reply:
Thanks Ian - see the new output from lmutil.exe at tests/data/prod-liense-v2.txt

I updated the tests to incorporate the updated output.